example/index.js

```diff
@@ -34,3 +34,8 @@ function cinque() {
   reset();
   ellipsis('.text p', 5);
 }
+
+function unoDifferentReplacementStr() {
```

Comment:
Referring to comments in `index.html`, this will be useful soon, but not in this PR.